Controllers/news.controllers.js

```diff
@@ -70,3 +71,13 @@ exports.postCommentToArticleId = (request, response, next) => {
       next(err);
     });
 };
+
+module.exports.patchArticleId = (request, response, next) => {
+  updateArticleById(request.body, request.params)
```

Comment:
I would like to see you save the req.body and req.params to variables. Your code needs to be as readable as possible.